kafka-ui-react-app/src/components/Dashboard/ClustersWidget/ClustersWidget.tsx

```diff
@@ -86,7 +86,14 @@ const ClustersWidget: React.FC<Props> = ({
           <tbody>
             {chunkItem.data.map((cluster) => (
               <tr key={cluster.name}>
-                <td>{cluster.name}</td>
+                <td>
+                  {cluster.readOnly && (
+                    <div className="tag has-margin-right is-info is-light">
```

Comment:
We're trying to use own styles with the styled-components and without classnames, aren't we?